### PR TITLE
cmd,internal,rpc: Implement openrpc.discover rpc method

### PR DESCRIPTION
**Pull request by meowsbits**
This implements a minimum-viable implementation to meet the
discovery method standard described by OpenRPC.

See commit message for more details.

### CODE REVIEW

**Comment by shanejonas** [comment]
More details [ECIP #1053](https://github.com/etclabscore/ECIPs/blob/master/ECIPs/ECIP-1053.md).

**Comment by meowsbits** [comment]
Waiting on upstream spellcheck fixes caught in the CI linter tests. Will need to update with revised JSON once those changes are resolved.

PR for those here -> https://github.com/etclabscore/ethereum-json-rpc-specification/pull/8.

File: internal/openrpc/openrpc_schema.go [hunk 1]
```diff
@@ -0,0 +1,1828 @@
...
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+
+package openrpc
+
+// This file contains a string constant containing the JSON schema data for OpenRPC.
+
+// OpenRPCSchema defines the default full suite of possibly available go-ethereum RPC
+// methods.
+const OpenRPCSchema = `
```
Comment (r8d8) [inline internal/openrpc/openrpc_schema.go:23]: Why can't we return data from a json file?

**Comment by tzdybal** [comment]
KISS to the extremum ;)
Returning from file requires extra tool for static resource bundling, which on turn should be extra build step

On May 13, 2019 3:50:21 PM GMT+02:00, Constantine <notifications@github.com> wrote:
>r8d8 commented on this pull request.
>
>
>
>> +//
>+// The go-ethereum library is distributed in the hope that it will be
>useful,
>+// but WITHOUT ANY WARRANTY; without even the implied warranty of
>+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
>+// GNU Lesser General Public License for more details.
>+//
>+// You should have received a copy of the GNU Lesser General Public
>License
>+// along with the go-ethereum library. If not, see
><http://www.gnu.org/licenses/>.
>+
>+package openrpc
>+
>+// This file contains a string constant containing the JSON schema
>data for OpenRPC.
>+
>+// OpenRPCSchema defines the default full suite of possibly available
>go-ethereum RPC
>+// methods.
>+const OpenRPCSchema = `
>
>Why can't we return data from a json file?
>
>-- 
>You are receiving this because your review was requested.
>Reply to this email directly or view it on GitHub:
>https://github.com/multi-geth/multi-geth/pull/76#pullrequestreview-236688399

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

File: internal/openrpc/openrpc_schema.go (hunk 1 above)
Comment (meowsbits) [reply internal/openrpc/openrpc_schema.go:23]: Yea, what [tzdybal said](https://github.com/multi-geth/multi-geth/pull/76#issuecomment-491833043); it's just more complicated that way... and we expect this string to be relatively extremely constant (set and forget), so extra mechanics to mediate the JSON as an external resource were expected to be of limited utility.

**Review by sorpaas — APPROVED** [review]

**Pull request merged**